python_modules/dagit/dagit/schema.py

```diff
@@ -80,6 +113,26 @@ def resolve_environment_type(self, _info):
         return Type.from_dagster_type(self._pipeline.environment_type)
 
 
+
+class Error(graphene.Interface):
+    message = graphene.String(required=True)
+    stack = graphene.NonNull(graphene.List(graphene.NonNull(graphene.String)))
+
+
+class PythonError(graphene.ObjectType):
+    class Meta:
+        interfaces = (Error, )
+
+    def __init__(self, exc_type, exc_value, exc_tb):
```

Comment:
if you know the types of these params, could you check.inst_param them. it's useful for documentation

Reply:
Hey! Looks like we'll need to create a new `class_param` which checks `isclass` instead of `isinstance` for exc_type, and exc_value is an `any` type (I think?) Honestly the sysinfo docs don't really indicate. Will add these.

Reply:
Hmm—looks like the `traceback` module doesn't actually export a `Traceback` class , and neither does `sys`. Not actually sure how to obtain the type to typecheck a sys_info traceback... May punt this one.

Reply:
Yeah if it's not immediately obvious don't go on a fishing expedition.